app/src/main/java/io/intrepid/contest/models/Entry.java

```diff
@@ -0,0 +1,12 @@
+package io.intrepid.contest.models;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class Entry {
+    public UUID id;
+    public String title;
+    public Date createdAt;
+    public Date updatedAt;
+    public String photoUrl;
```

Comment:
Yes, exactly, they'll come from Gson (there's no @SerializedName annotation because they're in the Gson converter format already)